content/projects/gsoc/2020/projects/machine-learning.adoc

```diff
@@ -70,6 +70,9 @@ The community bonding period is all about, well, community bonding. Rather than
 * Git integration for code segments (optional)
 * User documentation and examples
 
```

Comment:
Add links to changelog entries for phases? https://github.com/jenkinsci/machine-learning-plugin/blob/master/CHANGELOG.md

Reply:
+1 sounds like a good idea too!